spec/structuraid_core/design_codes/aci_318_19/rc/footings/bending_reinforcement_ratio_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'spec_helper'
+
+# rubocop:disable RSpec/FilePath
+RSpec.describe StructuraidCore::DesignCodes::ACI31819::RC::Footings::BendingReinforcementRatio do
+  describe '.call' do
+    subject(:result) do
+      described_class.call(
+        design_compression_strength:,
+        design_steel_yield_strength:,
+        width:,
+        effective_height:,
+        flexural_moment:,
+        capacity_reduction_factor:
+      )
+    end
+
+    let(:design_steel_yield_strength) { 420_000_000 } # N/(m**2)
+    let(:width) { 2.5 } # m
+    let(:effective_height) { 0.45 } # m
+    let(:capacity_reduction_factor) { 0.90 }
```

Comment:
I think our units system convention is:
```
mm for lengths
N for force
N/mm**2 (that is MPa) for stress and pressure
```

Are these specs and the formulae you coded compliant with those units?

Reply:
fuck, you are right, so flexural moment should be N*mm

Reply:
done!